wandb/sdk/launch/runner/local.py

```diff
@@ -108,15 +109,19 @@ def run(self, launch_project: LaunchProject) -> Optional[AbstractRun]:
         container_env: List[str] = container_inspect["ContainerConfig"]["Env"]
 
         if launch_project.docker_image is None or launch_project.build_image:
+            image_uri = construct_local_image_uri(launch_project)
             image = build_docker_image_if_needed(
                 launch_project=launch_project,
                 api=self._api,
                 copy_code=copy_code,
                 workdir=container_workdir,
                 container_env=container_env,
+                runner_type='local',
```

Comment:
Confused by this, won't this make it so the check for `gcp_vertex` (L220) in `build_docker_image_if_needed` never be true?

Reply:
this is in the local runner, this PR doesn't have the vertex runner in it — looks like i left the gcp line in `docker.py` in by accident lol you're right that it won't get hit in any of this code, i can take it out so the PR is cleaner

Reply:
Ah I see now.